build/webpack/webpack.client.config.js

```diff
@@ -2,10 +2,10 @@
 // Licensed under the MIT License.
 
 const common = require('./common');
```

Comment:
This had a loading error and from the docs didn't look relevant to ES6 module, so I removed:
https://www.npmjs.com/package/webpack-fix-default-import-plugin